site/_less/layouts/action-network-petitions/white-two-column.less

```diff
@@ -141,11 +145,106 @@
 
   .petition-info {
     @media screen and (min-width: @large-phone) {
-      padding: 0 1rem 0 0;
+      padding: 0 2rem 0 0;
     }
   }
 
+   [for="opt-in"] {
+     background-color: white;
+     color: @blue-800;
+     padding: 2rem 5px;
+  }
+
   .an-footer-cta {
     display: none;
   }
 }
+
+  .petition blockquote p{
+     font-size: 0.8em;
+     color: #ecedfb;
+    line-height: 2rem;
+  }
+ 
+   main.white-two-column h2{
+     margin-bottom: 0.5em;
+   }
+ 
+   .white-two-column [for="opt-in"]{
+     color: #8889a0;
+   }
+ 
+   .petition-info h2{
+     font-weight: 400;
+   }
+ 
+   .petition [type=email], .petition [type=text]{
```

Comment:
Should be combined with rule i noted at L163, also instead of targeting `[type="email"]` or `[type="text"]`, try using class names instead:

```less
.name, .address, .city, .email, .zip {}
```